### ISSUE TITLE
canvas is in the floating window, the content display is missing

### ISSUE THREAD

**Issue by jiajiala** [nothingislost/obsidian-hover-editor#292]
![image](https://github.com/user-attachments/assets/c32b5fc1-8ce2-4c57-9174-30174657659a)

As shown in the figure, when the left side is displayed in the suspended window, the title is missing, resulting in total ignorance of the main content of each card when used
Hope this BUG can be fixed, thanks

**Comment by pjeby** [comment]
Can you attach your canvas file?  I'm not seeing any issues like this with mine.  Please also check in sandbox with no themes/CSS snippets/other plugins to make sure it's not a CSS or plugin issue.  Also, report your Obsidian version, installer etc. from the "Show debug info" command in Obsidian.  Thanks.

**Comment by jiajiala** [comment]
[BBB.md](https://github.com/user-attachments/files/17788327/BBB.md)
![image](https://github.com/user-attachments/assets/83014d54-2750-4968-b69f-365edbf2d31b)
[Test1.canvas.md](https://github.com/user-attachments/files/17788332/Test1.canvas.md)
So these are the files, and you need to delete the .md

Here is an experiment in an empty library   without any CSS and other plugins
![9](https://github.com/user-attachments/assets/7b26faad-bf08-490e-9764-2d25b87cb4b7)
SYSTEM INFO:
	Obsidian version: v1.6.7
	Installer version: v1.6.7
	Operating system: Windows 10 Enterprise 10.0.19045
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: dark
	Community theme: none
	Snippets enabled: 17
	Restricted mode: off
	Plugins installed: 98
	Plugins enabled: 1
		1: Hover Editor v0.11.21

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by jiajiala** [comment]
I have just upgraded my obsidian to 1.7.6. the problem no longer appears and I will closeSorry to bother youBecause the obsidian of the new Ben has some plug-ins that are not available, I returned 1.6Thank you for this plug-in, which is the most important plug-in for me in obsidian

**Comment by pjeby** [comment]
Just so you know, the issue (which I am able to repro with your test files even on Obsidian 1.5.8), is that the Obsidian default style sheet hides `.inline-title` blocks when they are in a hover preview or embed.  This isn't a bug in Hover Editor per se, just a styling problem.  You can override the style with something like:

```css
.hover-popover.hover-editor .inline-title { display: block }
```
